src/test/java/dl/news/portal/domain/repository/NewsRepositoryTest.java

```diff
@@ -0,0 +1,80 @@
+package dl.news.portal.domain.repository;
```

Comment:
Don't test repositories, test services that use them. You should add services

Reply:
Done